jwala-webapp/src/main/webapp/resources/js/react/ResourcePane.js

```diff
@@ -166,7 +166,7 @@ var ResourcePane = React.createClass({
         }
     },
     displayDeployErrorMessage: function(response){
-        $.errorAlert(ResourcePane.parseDetailedErrorMsg(response, ResourcePane.DEFAULT_DEPLOY_ERR_MSG), "", undefined, response.responseJSON.content);
+        $.errorAlert(ResourcePane.parseDetailedErrorMsg(response, ResourcePane.DEFAULT_DEPLOY_ERR_MSG), "");
```

Comment:
Please revert to:

    $.errorAlert(ResourcePane.parseDetailedErrorMsg(response, ResourcePane.DEFAULT_DEPLOY_ERR_MSG), "", undefined, response.responseJSON.content);